### PR TITLE
Artillery

### PR DESCRIPTION
**Pull request by p1nga**
Initial Commit.
Folder Structure and M1A1 Collimator files.

### CODE REVIEW

File: addons/artillery/CfgVehicles.hpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class CfgVehicles
+{
```
Comment (commy2) [inline addons/artillery/CfgVehicles.hpp]: We put the { usually at the end of the line.

File: addons/artillery/CfgVehicles.hpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	// Made It A Search Light, For Initial Testing, Lets Change This....
+	class StaticSEARCHLight;
+	
+	class ACE_M1A1_Collimator_F: StaticSEARCHLight
+	{
+		displayName = "M1A1 Aiming Collimator";
+		model="\z\ace\addons\artillery\m1a1_collimator\M1A1_Collimator_F.p3d";
```
Comment (commy2) [inline addons/artillery/CfgVehicles.hpp]: model = PATHTOF(M1A1_Collimator_F.p3d);

**Comment by commy2** [comment]
Yay.

**Comment by SzwedzikPL** [comment]
Great to see that in ACE3 :D Any plans of system implementation @NouberNou ?

**Comment by p1nga** [comment]
Time for some proper testing of this,  probably have to tweak some of the model.cfg to get the right offset and size adjustment animations.

**Comment by p1nga** [comment]
I did a massive change to folder structure, and I think a few extra objects were added as well, I just didn't note them in the commit comment

**Comment by gienkov** [comment]
Is this ready for testing?

**Comment by Hammerr** [comment]
How are the things going? Will be soon anything ready for testing?

**Comment by NouberNou** [comment]
This is all still a ways off guys. :) Hold your horses. :D

**Pull request closed without merging**